pytests/tests/test_downgrade.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2019 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the GNU General Public License v2 (the "License");
+# you may not use this file except in compliance with the License. The terms
+# of the License are located in the COPYING file of this distribution.
+#
+
+import pytest
+
+
+@pytest.fixture(scope='module', autouse=True)
+def setup_test(utils):
+    yield
+    teardown_test(utils)
+
+
+def teardown_test(utils):
+    utils.run(['tdnf', 'erase', '-y', 'tdnf-test-two', 'tdnf-test-multiversion'])
+
+
+def test_downgrade_no_arg(utils):
+    ret = utils.run(['tdnf', 'downgrade'])
+    assert(ret['retval'] == 1011)
```

Comment:
why do we expect a non zero return code here? Downgrade command should pass.

Reply:
I ended up fixing the test tear down part of various scripts, still in progress, will push the proper code soon.

Reply:
Everything is taken care now and all tests are passed.